charts/crowdsec/values.schema.json

```diff
@@ -903,6 +906,73 @@
                 "listen_port",
                 "webhook_path"
             ]
+        },
+        "VictoriaLogsSource": {
+            "type": "object",
+            "description": "https://docs.crowdsec.net/docs/next/data_sources/victorialogs",
+            "properties": {
```

Comment:
There is `since` property missing no ? (from your [PR](https://github.com/crowdsecurity/crowdsec/pull/3310/files#diff-d43405ff374ab2d76c37f3971671e5c97078d77f5830277b91a7c2c5d3ed1d16R43))

Reply:
Thanks, fixed in https://github.com/crowdsecurity/helm-charts/pull/233/commits/2c3306fe599c6edcaabfc4f22752122b17a24801